icu4c/source/test/intltest/units_test.cpp

```diff
@@ -1238,23 +1240,35 @@ void UnitsTest::testUnitsConstantsDenomenator() {
         {"portion-per-1E5", 100000},
         {"portion-per-1e6", 1000000},
         {"portion-per-1E6", 1000000},
-        // ICU-22781: {"portion-per-1e10", 10000000000},
-        // ICU-22781: {"portion-per-1E10", 10000000000},
-        // ICU-22781: {"portion-per-1e18", 1000000000000000000},
-        // ICU-22781: {"portion-per-1E18", 1000000000000000000},
+        {"portion-per-1e10", 10000000000},
+        {"portion-per-1E10", 10000000000},
+        {"portion-per-1e18", 1000000000000000000},
+        {"portion-per-1E18", 1000000000000000000},
+
         // Test for constant denominators that are randomly selected.
-        // ICU-22781: {"liter-per-12345-kilometer", 12345},
-        // ICU-22781: {"per-1000-kilometer", 1000},
-        // ICU-22781: {"liter-per-1000-kiloliter", 1000},
+        {"liter-per-12345-kilometer", 12345},
+        {"per-1000-kilometer", 1000},
+        {"liter-per-1000-kiloliter", 1000},
+
         // Test for constant denominators that give 0.
         {"meter", 0},
         {"meter-per-second", 0},
         {"meter-per-square-second", 0},
-        // NOTE: The following constant denominator should be 0. However, since
+
+        // NOTE: The following constant denominator should be 1e9. However, since
+        // `portion-per-1e9` is treated as a unit in CLDR,
+        // the unit does not have a constant denominator.
+        // This issue should be addressed in CLDR & ICU.
+        {"portion-per-1000000000", 0},
```

Comment:
Is there an outstanding ticket for this?  If so, you should probably include the ticket number (or a link) here.

Reply:
`portion-per-1e9` is treated as a _precomputed form_ for translation, not a single unit.

[Some units already have 'precomputed' forms, such as **kilometer-per-hour**; where such units exist, they should be used in preference.](https://unicode.org/reports/tr35/tr35-general.html#Unit_Identifier_Uniqueness:~:text=Some%20units%20already%20have%20%27precomputed%27%20forms%2C%20such%20as%20kilometer%2Dper%2Dhour%3B%20where%20such%20units%20exist%2C%20they%20should%20be%20used%20in%20preference.)

We shouldn't test for the wrong behavior. 
That being said, given the schedule, and the fact that we are going fix the problem fully in 48, it's ok for `{"portion-per-1000000000", 1000000000},` be present but commented out.

Reply:
Sure, I have added it back with `logKnownIssue` so it can be addressed in the next release.